juriscraper/pacer/appellate_attachment_page.py

```diff
@@ -156,11 +164,44 @@ def _get_page_count_from_tr(row: html.HtmlElement) -> Optional[int]:
         :param row: Table row as an lxml element
         :return: Attachment page count
         """
+        input_els = row.xpath(".//input[@class='selDocCl']")
+        for input_el in input_els:
+            try:
+                onclick = input_el.xpath("./@onclick")
+                if onclick and "sumDocSelected" in onclick[0]:
+                    sum_doc_selected_parts = reverse_sumDocSelected_function(
+                        onclick[0]
+                    )
+                    if sum_doc_selected_parts:
+                        return sum_doc_selected_parts["page_count"]
+            except IndexError:
+                continue
+
```

Comment:
Since this block of code is quite similar to the one in `_get_file_size_bytes_from_tr`, could we place it in a common place to avoid duplicating code? Perhaps we could create an additional helper method in `pacer.utils` that takes the key you want to retrieve either `page_count` or `file_size_bytes` as an argument?